src/commands/lambda/instrument.ts

```diff
@@ -149,9 +150,29 @@ export class InstrumentCommand extends Command {
       return
     }
 
-    const flushMetricsToLogs = this.flushMetricsToLogs ?? this.config.flushMetricsToLogs ?? true
+    if (!['true', 'false', undefined].includes(this.flushMetricsToLogs?.toLowerCase())) {
+      this.context.stdout.write('No valid value specified for flushMetricsToLogs.\n')
+
+      return
+    }
+
+    if (!['true', 'false', undefined].includes(this.tracing?.toLowerCase())) {
+      this.context.stdout.write('No valid value specified for tracing.\n')
+
+      return
+    }
+
+    const flushMetricsToLogs = this.flushMetricsToLogs
```

Comment:
A nice way to test it could be to extract this logic a pure function taking two parameters (two strings representing flushMetricsToLogs et config.flushMetricsToLogs) then apply all for possible inputs and assert the result.
This method could be reused for tracing since it seems that it's the exact same logic